catalog/log-export/org/logbucket-export/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: logbucket-export
+  annotations:
+    blueprints.cloud.google.com/title: Cloud Logging Log Bucket Export blueprint
```

Comment:
We should local anno in Kptfiles and fnconfigs - `config.kubernetes.io/local-config: "true"` due to https://github.com/GoogleCloudPlatform/blueprints/issues/174

Reply:
Ack